ssh.el

```diff
@@ -260,6 +260,10 @@ how ssh X display tunelling interacts with frames on remote displays."
                 (t
                  (cd-absolute (concat comint-file-name-prefix "~/"))))
         (error nil)))))
+
+  ;; enable tab-complete in remote shell
+  (setq default-directory (concat "/ssh:" input-args ":~/"))
```

Comment:
This should probably use `tramp-default-method` or similar instead of hard-coding "ssh".